biojava-integrationtest/src/test/java/org/biojava/nbio/structure/test/io/cif/CifFileSupplierImplTest.java

```diff
@@ -0,0 +1,176 @@
+package org.biojava.nbio.structure.test.io.cif;
+
+import org.biojava.nbio.structure.*;
```

Comment:
We are trying to avoid star (*) imports. Could you try to replace them for specific Class imports? It happens in a number of cases.

Reply:
Package imports are removed in 0b59a10